CodePen.js

```diff
@@ -83,9 +83,10 @@ function createStylableSelect(simulatedSelect, optionsData = []) {
     })
     .on("focusout", function (e) {
       const target = $(e.target);
+      const isWrapper = target.is(".simulated-select-options-wrapper");
       const isOption = target.is('[role="option"]');
       const isOptionsEnd = target.is(".simulated-select-options-end");
-      if (!isOption && !isOptionsEnd) {
+      if (!isWrapper && !isOption && !isOptionsEnd) {
```

Comment:
The bug happened after adding the `focusout` event listener, but instead of revertingremoving it, adding a check for `!isWrapper` got it working again.